src/Raven.Server/Documents/Sharding/ShardLocator.cs

```diff
@@ -28,7 +28,7 @@ public static DocumentIdsByShardIndex GroupIdsByShardIndex(IEnumerable<Slice> id
             for (var i = 0; i < ids.Count; i++)
             {
                 var id = ids[i];
-                var shardId = shardedContext.GetShardId(context, id);
+                var shardId = ShardedContext.GetShardId(context, id);
                 var index = shardedContext.GetShardIndex(shardId);
```

Comment:
let's use the `shardedContext.GetShardIndex(context, id)`

Reply:
done